config/config.yaml

```diff
@@ -4,3 +4,10 @@
 imports:
     - { resource: '@SyliusPriceHistoryPlugin/config/events.yaml' }
     - { resource: '@SyliusPriceHistoryPlugin/config/grids/*' }
+        
+framework:
+    messenger:
+        buses:
+            sylius.no_transaction_command_bus:
```

Comment:
This command bus doesn't have `doctrine_transaction` middleware. It's a workaround for a problem with an infinite loop in the "on flush" event listener. It's not a good fix. I did it only to make tests green and to see if something more fails.